app/controllers/assessor_interface/timeline_events_controller.rb

```diff
@@ -0,0 +1,15 @@
+module AssessorInterface
+  class TimelineEventsController < BaseController
+    def index
+      @application_form =
+        ApplicationForm.includes(:timeline_events).find(
+          params[:application_form_id]
+        )
+
+      @timeline_events =
```

Comment:
Should we get the timeline events from the `@application_form` given we included the data in that query?

Reply:
Ah, I need to remove the `includes(:timeline_events)` actually, I discovered that due to the `created_at: :desc` it executes another query anyway, and I figured two queries for this page isn't too bad (it's not an N+1 issue).